classes/app.rb

```diff
@@ -0,0 +1,120 @@
+require_relative '../modules/appfunctions'
+require 'pry'
```

Comment:
- [OPTIONAL] Please kindly add `pry` to your `Gemfile`.

Reply:
Added.

Thanks a lot.